service/src/main/resources/config/aou/test/SC2023Q3R1/sql/t_device.sql

```diff
@@ -0,0 +1,19 @@
+SELECT DISTINCT c.concept_id, c.concept_name, c.concept_code,
+                CASE
+                    WHEN c.standard_concept IS NULL THEN 'Source'
+                    WHEN c.standard_concept = 'S' THEN 'Standard'
+                    ELSE 'Unknown' END standard_concept,
+                v.vocabulary_name,
+                v.vocabulary_id
+
+FROM `all-of-us-ehr-dev.SC2023Q3R1.device_exposure` AS de
```

Comment:
Here you're selecting all the device types that have >=1 occurrence row, instead of selecting just all device types. Is that what you intended? Or what was the motivation behind the changes to the device entity?

Reply:
In Aou we do not show *all* whe show only those device or any other domain concept_ids only if there is atleast 1 obeservation or item_count or rollup count is >=1 This part is from aou's indexing build SQL: [build-cb-criteria-device.sh](https://github.com/all-of-us/workbench/blob/a909b2097bfa407c050fdaa5b2b351a212f4f8c1/api/db-cdr/generate-cdr/build-cb-criteria-device.sh#L67-L71)